test/tools/unified-spec-runner/runner.ts

```diff
@@ -113,6 +113,9 @@ async function runUnifiedTest(
       return ctx.skip();
     }
 
+    const ping = await utilClient.db().admin().command({ ping: 1 });
```

Comment:
Huh, I'm surprised this has the intended effect.  My understanding is that the runner has to get the cluster time from _after_ all collections are dropped, created, and initial data has been added, whereas this is happening before that.

Reply:
Oh yeah, I was playing around with the timing of it but moved it back to after initial data was created. Surprisingly it did work taking the cluster time before initial data - maybe the transactions just needed any cluster time set in the past?